README.md

```diff
@@ -21,18 +36,18 @@ You can obtain CUDA from [https://developer.nvidia.com/cuda-downloads](https://d
 
 ### Get dependencies
 
-**Note**: This repo uses submodules. Make sure you cloned recursively:
+#### CMake
 
+Install the latest version of `cmake` using `pip`:
 ```bash
-git clone --recurse-submodules git@github.com:gpuopenanalytics/libgdf.git
+pip install cmake
```

Comment:
cmake is also on conda-forge

Reply:
I think we should give both options as there will be users installing libgdf and pygdf from source without using conda.

Reply:
That is true, and it is mentioned further down in the doc about using a conda environment that will install cmake and arrow for you. 

The issue with the conda process is:

1. it is at the end of the README and if that is the preferred path for build and test then it should be upfront first
2. the build environment that is defined in `conda_environments/dev_py35.yml` is outdated and has not been updated to the same standard as the information on the README or what is used in the build environments

Given those reasons I stuck with focusing on installing without conda, and welcome input on fixing the other problem of defining preferred means to build and test libgdf.

Additionally Ubuntu doesn't support any CMake version except v3.5 so pip is one way to get around the trouble of installing the latest version and being operating system independent.

Reply:
> it is at the end of the README and if that is the preferred path for build and test then it should be upfront first

I think we should have conda as the recommended way of installing dependencies, followed by pip, followed by apt/yum/etc installs

> the build environment that is defined in conda_environments/dev_py35.yml is outdated and has not been updated to the same standard as the information on the README or what is used in the build environments

Yes, this needs to be addressed, and if it's addressed we can have at the top of the readme they can create a conda environment from the file and it automagically takes care of everything.